dsl/api.go

```diff
@@ -150,6 +150,30 @@ func License(fn func()) {
 	eval.IncompatibleDSL()
 }
 
+// Randomiser sets the API example randomiser.
```

Comment:
Just to be consistent with the rest of the code base (which uses the American based syntax) could we use `Randomizer`?